ipatests/test_integration/test_installation.py

```diff
@@ -2123,3 +2123,79 @@ def test_hostname_matching_domain(self):
         assert result.returncode == 1
         assert 'hostname cannot be the same as the domain name' \
             in result.stderr_text
+
+
+class TestNsslapdIgnoreTimeSkew(IntegrationTest):
+    """
+    Test to check nsslapd-ignore-time-skew is not disabled.
+    """
+    num_replicas = 1
+    topology = 'line'
+
+    def install(self, mh):
```

Comment:
Do you really need to override this method? I think you can remove it and rely on the default `install` method from `IntegrationTest` class.